docs/throwing-exceptions.md

```diff
@@ -38,3 +38,22 @@ A.CallTo(() => fakeShop.NumberOfSweetsSoldOn(A<DateTime>._))
                                                 " is closed on " +
                                                 callObject.Arguments[0]));
 ```
+
+## Throwing exceptions from an async method
+
+When a method returns a `Task` or `Task<T>`, there are two ways it can indicate
+failure via an exception:
+
+- throw the exception synchronously, i.e. not actually return a `Task`
+- "throw asynchronously", i.e. return a failed task with the exception.
+
+The former is supported by the `Throws` method described above, in the same way as if the
+method was synchronous. The latter can be configured by using the `ThrowsAsync` method:
+
+```csharp
+A.CallTo(() => fakeShop.OrderSweetsAsync("cheeseburger"))
+ .ThrowsAsync(new ArgumentException("'cheeseburger' isn't a valid sweet category"));

```

Comment:
I direct you to http://www.worksburger.com/Home.aspx

![image](https://cloud.githubusercontent.com/assets/3275797/21080503/cb1c540c-bf7e-11e6-93e4-9607847442b7.png)

You still want to keep your example?

(As an aside, I have actually eaten (half of) this thing. It's better than you'd think. A little sweet. It definitely did open my eyes to "peanut butter on burgers", which is quite good!)